dist/build.spec

```diff
@@ -88,6 +88,13 @@ Requires:       %{__pkg_name}-mkbaselibs
 Recommends:     %{__pkg_name}-mkdrpms
 %endif
 
+# With fedora 33 the POSIX module was split out of the perl
+# package
+%if 0%{?fedora_version} >= 33
+BuildRequires: perl(POSIX)
+Requires: perl(POSIX)
+%endif
+
```

Comment:
Wouldn't it make sense to always require it? The perl packages should have provides for the intree copies of modules.